elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -264,6 +266,20 @@ export abstract class BaseTransaction {
 			errors.push(idError);
 		}
 
+		/* tslint:disable */
+		if (this.type !== (<typeof BaseTransaction>this.constructor).TYPE) {
```

Comment:
```
ERROR: (no-angle-bracket-type-assertion) elements/lisk-transactions/src/base_transaction.ts[269, 22]: Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
```

Reply:
@shuse2 @lsilvs

Reply:
try
`(this.constructor as typeof BaseTransaction)`